[2023-09-18 11:36:00,121][ignite.engine.engine.Engine][INFO] - Engine run starting with max_epochs=1.
[2023-09-18 11:36:03,533][ignite.engine.engine.Engine][ERROR] - Current run is terminating due to exception: index out of range in self
[2023-09-18 11:36:03,617][ignite.engine.engine.Engine][ERROR] - Engine run is terminating due to exception: index out of range in self
